core/src/consumer.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
Remove this entire file from this PR to prevent unnecessary merge conflicts later on